level.go

```diff
@@ -0,0 +1,39 @@
+package lorg
+
+// Level describes all available log levels for log records.
+type Level int
+
+const (
+	// LevelFatal should be used for messages with FATAL severity.
+	LevelFatal Level = iota
+
+	// LevelError should be used for messages with ERROR severity.
+	LevelError
+
+	// LevelWarning should be used for messages with WARNING severity.
+	LevelWarning
+
+	// LevelInfo should be used for messages with INFO severity.
+	LevelInfo
+
+	// LevelDebug should be used for messages with DEBUG severity.
+	LevelDebug
+)
+
+// String returns the string representation of a given logging level.
+func (level Level) String() string {
+	switch level {
+	case LevelFatal:
+		return "FATAL"
+	case LevelError:
+		return "ERROR"
+	case LevelWarning:
+		return "WARNING"
+	case LevelInfo:
+		return "INFO"
+	case LevelDebug:
+		return "DEBUG"
+	}
+
+	return "UNKNOWN"
```

Comment:
I guess variant without default should be more flexible than with, because if somebody adds new logic (case letters, for example)  for work with string representation of Level than he have to add `return` at the end of function, because Go compiler will warn.

readability it's subjective thing, if we want to be more effective (rational), than we should use objective properties - code complexity, for example, so your case is objectively more complex:

```
 λ  ~/go/src/github.com/kovetskiy/lorg dev✗ cat level.go
package lorg

// Level describes all available log levels for log records.
type Level int

const (
        // LevelFatal should be used for messages with FATAL severity.
        LevelFatal Level = iota

        // LevelError should be used for messages with ERROR severity.
        LevelError

        // LevelWarning should be used for messages with WARNING severity.
        LevelWarning

        // LevelInfo should be used for messages with INFO severity.
        LevelInfo

        // LevelDebug should be used for messages with DEBUG severity.
        LevelDebug
)

// String returns the string representation of a given logging level.
func (level Level) String() string {
        switch level {
        case LevelFatal:
                return "FATAL"
        case LevelError:
                return "ERROR"
        case LevelWarning:
                return "WARNING"
        case LevelInfo:
                return "INFO"
        case LevelDebug:
                return "DEBUG"
        }

        return "UNKNOWN"
}
 λ  ~/go/src/github.com/kovetskiy/lorg dev✗ gocyclo -avg level.go
6 lorg (Level).String level.go:24:1
Average: 6
```

``` λ  ~/go/src/github.com/kovetskiy/lorg dev✗ cat level.go
package lorg

// Level describes all available log levels for log records.
type Level int

const (
        // LevelFatal should be used for messages with FATAL severity.
        LevelFatal Level = iota

        // LevelError should be used for messages with ERROR severity.
        LevelError

        // LevelWarning should be used for messages with WARNING severity.
        LevelWarning

        // LevelInfo should be used for messages with INFO severity.
        LevelInfo

        // LevelDebug should be used for messages with DEBUG severity.
        LevelDebug
)

// String returns the string representation of a given logging level.
func (level Level) String() string {
        switch level {
        case LevelFatal:
                return "FATAL"
        case LevelError:
                return "ERROR"
        case LevelWarning:
                return "WARNING"
        case LevelInfo:
                return "INFO"
        case LevelDebug:
                return "DEBUG"
        default:
                return "UNKNOWN"
        }
}
 λ  ~/go/src/github.com/kovetskiy/lorg dev✗ gocyclo -avg level.go
7 lorg (Level).String level.go:24:1
Average: 7```

wow